sass/components/_emoticons.scss

```diff
@@ -309,11 +309,11 @@
 
     &.selected,
     &:hover {
-        background-color: v(center-channel-color-15);
+        background-color: v(center-channel-color-16);
     }
 
     &:active {
-        background-color: v(center-channel-color-30);
+        background-color: v(center-channel-color-32);
```

Comment:
CSS variables were no longer available, and so just pick the closest possible values - https://github.com/mattermost/mattermost-webapp/blob/master/utils/utils.jsx#L894-L915.  Adding @asaadmahmood for review, thanks

Reply:
Yes, that is correct.